dev/devicelab/bin/tasks/windows_app_template_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:io';
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/task_result.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+import 'package:path/path.dart';
+
+const int _kLicenseLines = 4;
+const List<String> _kFilesToMatch = <String>[
```

Comment:
The files that are expected to always be identical between the template and the integration tests. Some files, e.g. `flutter_window.cpp` are expected to be different, as they may contain test-specific code, such as registering custom method handlers.